dev/MRTCore/packaging/MrtCore.PriGen.targets

```diff
@@ -22,6 +22,15 @@
     <DefaultLanguage Condition="'$(DefaultLanguage)' == ''">en-US</DefaultLanguage>
   </PropertyGroup>
 
+  <!--
+    PRIResource is not by default in the list of Build Actions for WPF apps. So add it!
+    If we don't add it, our attempt to by default include RESW files in PRIResource will
+    fail (VS will inject MSBuild code in the project file to undo that include).
+  -->
+  <ItemGroup>
+    <AvailableItemName Include="PRIResource" Condition="'$(UseWPF)' == 'true'"/>
```

Comment:
should we always include PRIResource if it's not? WPF is what we found to have this issue, but not something we need to be limited to.

Reply:
let's always include. i'll test with non-WPF also to verify it doesn't break anything.

Reply:
done